toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -599,6 +602,7 @@ func packSRPMs(specStates []*specState, distTag, buildDir string, templateSrcCon
 		if result.err != nil {
 			logger.Log.Errorf("Failed to pack (%s). Error: %s", result.specFile, result.err)
 			err = result.err
+			logger.Log.Warn("Cancelling outstanding workers")
```

Comment:
Why not just include this message inside the error above since it's part of the same "event"? For instance:
`logger.Log.Errorf("Failed to pack (%s). Cancelling outstanding workers. Error: %s", result.specFile, result.err)`

Reply:
Was getting lost in all the log spam, but that spam is fixed now so no reason not to unify.